_config.yml

```diff
@@ -49,6 +49,7 @@ alias:
   why-cypress/: guides/overview/why-cypress.html
 
   api/index.html: api/introduction/api.html
+  api/assertions/assertions.html: guides/references/assertions.html
```

Comment:
🤔 It feels a bit awkward to have to use an explicit redirect. If there's a less awkward way to do this then lemme know.

Reply:
I think this is the way to do it, but yes, it is weird.